samples/nrf9160/download/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _download_sample:
+
+nRF9160: Download sample
+########################
+
+The Download sample demonstrates how to download a file from a HTTP or CoAP server using the download_client library, with optional TLS or DTLS.
+
+Overview
+********
+
+The sample first initializes the :ref:`nrfxlib:bsdlib` and AT communications.
+Next, it provisions a certificate to the modem using the :ref:`modem_key_mgmt` library if :option:`CONFIG_SAMPLE_SECURE_SOCKET` is set.
+The certificate file name and security tag can be configured via :option:`CONFIG_SAMPLE_SEC_TAG` and :option:`CONFIG_SAMPLE_CERT_FILE`, respectively
+Provisioning must be done before connecting to the LTE network, because certificates can only be provisioned when the device is not connected.
+
+The sample then establishes a connection to the LTE network, sets up (optionally) secure socket options, and uses the download_client library to download a file from a HTTP server.
+
+
+Downloading from a CoAP server
+==============================
+
+To enable CoAP block-wise transfer, it is necessary to enable Zephyr's CoAP stack via :option:`CONFIG_COAP`.
```

Comment:
We still haven't found any reference CoAP server to download file from, haven't we?

Reply:
Nope :(